qds_sdk/commands.py

```diff
@@ -154,12 +154,13 @@ def get_jobs_id(cls, id):
         return r.text
 
 
-    def get_results(self, fp=sys.stdout, inline=True, delim=None):
+    def get_results(self, fp=sys.stdout, inline=True, delim=None, fetch=True):
         """
         Fetches the result for the command represented by this object
 
         Args:
             `fp`: a file object to write the results to directly
+            `fetch`: fetches the result if it is small, else returns the s3 bucket location
```

Comment:
This statement needs improvement.
The results are always fetched when they are small. When the results are big, they would be fetched if this is true. Otherwise the s3 location would be returned.